composer.json

```diff
@@ -13,7 +13,8 @@
         "php": ">=5.4 || ^7.0",
         "sokil/php-mongo": "1.*",
         "symfony/console": ">=2.0 <5.0",
-        "symfony/yaml": ">=2.0 <5.0"
+        "symfony/yaml": ">=2.0 <5.0",
+        "alcaeus/mongo-php-adapter": "^1.1"
```

Comment:
This lib requires new mongo driver, we still support old adapter. Support of new adapter documented in https://github.com/sokil/php-mongo-migrator#compatibility-with-php-7. You can't directly require this, only through suggestion. And adapter already suggested my odm

Reply:
okay